examples/js/renderers/SoftwareRenderer.js

```diff
@@ -713,7 +713,8 @@ THREE.SoftwareRenderer = function ( parameters ) {
 				mpUV12 = new THREE.Vector2(),
 				mpUV23 = new THREE.Vector2(),
 				mpUV31 = new THREE.Vector2(),
-				tempFace = { vertexNormalsModel : [] };
+				tempFace = { vertexNormalsModel : []
+							, color : { r: face.color.r, g: face.color.b, b: face.color.b } };
```

Comment:
@daoshengmu you could perhaps just do color: face.color, since it is not modified, therefor saving some memory?